content/en/docs/zero-code/python/_index.md

```diff
@@ -45,6 +45,13 @@ Running `opentelemetry-bootstrap` without arguments lists the recommended
 instrumentation libraries to be installed. For more information, see
 [`opentelemetry-bootstrap`](https://github.com/open-telemetry/opentelemetry-python-contrib/tree/main/opentelemetry-instrumentation#opentelemetry-bootstrap).
 
+{{% alert title="Using `uv`?" color="warning" %}} If you are using the
```

Comment:
Oops, apparently, the title processes HTML rather than markdown, so this needs to be:
```suggestion
{{% alert title="Using <code>uv</code>?" color="warning" %}} If you are using the
```